### PR TITLE
Improve documentation

### LINKED ISSUE #306: Make sure frequency analysis functions are included in the docs
* xclim version: 0.12.2

### Description

At the moment the frequency analysis functions are not very visible in the docs.

### PR DESCRIPTION
**Pull request by huard**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated

### After merging to master and before closing the branch:
- [ ] bumpversion (minor / major / patch) has been called on `master` branch
- [ ] Tags have been pushed

* **What kind of change does this PR introduce?** 
Documentation update
  - Display example notebook
  - Clarify indices vs indicator instances
  - Display list of indicators

* **Does this PR introduce a breaking change?** 
No

* **Other information**:

This PR fixes #306, #333

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,30 +1,12 @@
-======
-xclim
-======
+===================================
+xclim: Climate indices computations
+===================================
 
-.. image:: https://github.com/Ouranosinc/xclim/blob/master/_static/_images/xclim-logo.png
```
Comment (Zeitsperre) [inline README.rst]: No love for the logo?

Comment (huard) [reply README.rst]: I moved it to the side-bar in RTD. I thought it occupied a lot of real estate.

Comment (Zeitsperre) [reply README.rst]: Ah, OK. I agree, it is a bit in your face. I'm just so used to seeing it, haha.

Comment (huard) [reply README.rst]: Not closed to get it back up if you can manage to have it scaled down, possibly inline in the title.

Comment (Zeitsperre) [reply README.rst]: I could quickly look into that.

Comment (Zeitsperre) [reply README.rst]: Trying out a manually rescaled image. GitHub has eliminated any means of doing rescaling via restructuredtext unfortunately.

**Comment by huard** [comment]
I don't think you committed the small logo.

**Pull request merged**